plugins/woocommerce/package.json

```diff
@@ -807,12 +808,13 @@
 		"build:project:actualize-translation-domains": {
 			"command": "node ./bin/package-update-textdomain.js",
 			"files": [
-				"packages/**/*.php"
+				"packages/action-scheduler/action-scheduler.php",
+				"packages/woocommerce-admin/woocommerce-admin.php"
 			],
 			"output": []
 		},
 		"watch:build:project:copy-assets": {
-			"command": "nodemon --exec \"pnpm run\" build:project",
+			"command": "nodemon --exec \"pnpm '/^build:project:copy-assets:.*$/'\"",
```

Comment:
I'm worried that this might be a little too brittle since it relies on the asset copy operation always being `build:project:copy-assets:.*` I _do_ like the premise, however, how would you feel about checking `NODE_ENV` instead and only doing this in a production build?

Reply:
Well, watch, by definition, is a non-production context, so I'm intentionally narrowing this down.
`NODE_ENV` would be added complexity; it would be to revert to `build:project` - changes on lines 811-812 making sure `wireit` will doing minimal work anyway,